android/app/src/main/kotlin/com/jabook/app/jabook/audio/processors/LufsAnalysisWorker.kt

```
@@ -0,0 +1,293 @@
+                            LufsRmsEstimator.estimateLufsFromPcm16(
+                                pcm16Data = pcmBuffer.copyOf(samplesRead),
+                                channels = channels,
+                            )
+                        if (estimate != null) {
+                            estimates.add(estimate)
+                        }
+                    }
+                }
+
+                LufsRmsEstimator.integrateEstimates(estimates)
+            } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)